e2e-tests/spring-examples/target/classes/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
```

Comment:
why are you adding the content of "target" folder from `spring-examples` (which does not exist any more) into Git?

Reply:
Because: when i did git status , i got : Untracked files:
  (use "git add <file>..." to include in what will be committed)
        e2e-tests/spring-examples/